scripts/apps/authoring/authoring/services/AuthoringService.js

```diff
@@ -418,7 +418,9 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
         if (self.isPublished(currentItem)) {
             // if not the last published version
             if (angular.isDefined(item.archive_item) &&
-                item._current_version !== item.archive_item._current_version && item.operation !== 'move') {
+                item._current_version !== item.archive_item._current_version &&
+                (item.operation == null || item.operation !== 'move') &&
```

Comment:
Why do you compare to `null`? Is there a case where `(item.operation == null || item.operation !== 'move')` and `(item.operation !== 'move')` produce different results?

Reply:
changed

Reply:
Thanks. I'm wondering why the checks have to be here in the first place. If I understand correctly, the function `itemActions` should return which actions are available on the item which in turn are displayed [here](https://user-images.githubusercontent.com/2076953/48822035-39617180-ed64-11e8-8d6b-2952c56307e1.png). What does it have to do with the operation?

Reply:
Somehow the condition tried to detect if it is the last published version for item.
Seams that there is already a field for this, 'last_published_version' so I used it.